tsdownsample/downsampling_interface.py

```diff
@@ -345,10 +346,11 @@ def downsample(
         self,
         *args,  # x and y are optional
         n_out: int,
-        parallel: bool = False,
+        n_threads: int = 1,
         **kwargs,
     ):
         """Downsample the data in x and y."""
+        parallel = n_threads > 1
         return super().downsample(*args, n_out=n_out, parallel=parallel, **kwargs)
```

Comment:
Do I see correctly that `n_threads` does not get passed to the Rust bindings of the downsampling functions?

Reply:
Probably because the macros require an update!